controllers/users-controllers.js

```diff
@@ -141,8 +138,8 @@ const signup = async (req, res, next) => {
   try {
     token = jwt.sign(
       { userId: createdUser.id, email: createdUser.email },
-      process.env.jwt_secret,
-      { expiresIn: "1h" }
+      process.env.jwt_secret
+      // { expiresIn: "1h" }
```

Comment:
Why the comment, also i think its better not to use the default expiration since someone else that will want to know the expiration will need to go to jwt documentation and check what it is, also if they change it it will be changed for us as well.
its better to use our own expiration and use a constant for it so it will be clear